cloud_dashboard/src/i18n.ts

```diff
@@ -1,6 +1,7 @@
 import i18n from "i18next";
```

Comment:
@RyoYamashita765 This can be:

```suggestion
import i18n from 'i18next';
```

Reply:
@naoi  In other parts of the import, double quotes were used. What would be the reason for using single quotes in this section only?